playbooks/aws/openshift-cluster/launch.yml

```diff
@@ -32,6 +32,13 @@
       cluster: "{{ cluster_id }}"
       type: "{{ k8s_type }}"
 
+  - include: ../../common/openshift-cluster/set_infra_launch_facts_tasks.yml
+  - include: tasks/launch_instances.yml
+    vars:
+      instances: "{{ infra_names }}"
+      cluster: "{{ cluster_id }}"
+      type: "{{ k8s_type }}"
+
```

Comment:
would it make more sense to add an additional variable here for node_type instead of having a separate playbook for setting the launch facts?